### PR TITLE
Update validator guide

### LINKED ISSUE #1793: Validator guide misses step to enable generator methods
### Description
The node settings were changed to improve node security, and this results in an additional step that validators need to perform to enable block generation on their node.

In the node config, they need to enable methods in the `rpc` section.

Some additional issues:

- generator status can not be requested directly after enabling forging (only after first block was generated)
- add note: generating hash onion can result in error, but the hash onion is still created successfully

### Affected branch / URL
`development`

### Additional resources
https://github.com/LiskHQ/lisk-sdk/pull/8478 will hcange the config option from `disabledMethods` to `enabledMethods`

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #1793

### CODE REVIEW

File: docs/modules/ROOT/pages/run-blockchain/become-a-validator.adoc [hunk 1]
```diff
@@ -382,6 +407,21 @@ Sidechain node::
...
+----
+
+This suggests that the API timed out.
+However, the hash onion is still created in most cases.
+You can wait for a while and then <<get-the-hash-onion>>, to verify that it actually is set.
+====
+
 === Get the hash onion
 
 Currently, there is nothing else to do with the hash onion, after it is created.
```
Comment (chris529) [inline docs/modules/ROOT/pages/run-blockchain/become-a-validator.adoc:427]: imho, I would suggest slightly rewording it as follows:
"Once the hash onion is created, there are no further actions or tasks associated with it."

File: docs/modules/ROOT/pages/run-blockchain/enable-block-generation.adoc [hunk 2]
```diff
@@ -51,12 +51,16 @@ It consists of the following three properties:
 
 WARNING: Don't trust <<validator-info-data>> from public APIs or explorers, make sure to always use your local data!
```
Comment (chris529) [inline docs/modules/ROOT/pages/run-blockchain/enable-block-generation.adoc]:
```suggestion
WARNING: Don't trust any <<validator-info-data>> from public APIs or explorers, make sure to always use your local data!
```

**Review by chris529 — APPROVED** [review]
LGTM! I just left a couple of comments 👍

File: docs/modules/ROOT/pages/run-blockchain/become-a-validator.adoc [hunk 3]
```diff
@@ -382,6 +407,21 @@ Sidechain node::
 
 To verify that the hash onion was set correctly, <<get-the-hash-onion>> back from the node.
 
+[CAUTION]
+====
+If you see the following error after running the command:
```
Comment (shuse2) [inline docs/modules/ROOT/pages/run-blockchain/become-a-validator.adoc:412]: I updated the endpoint on https://github.com/LiskHQ/lisk-sdk/pull/8521, so this error shouldn't happen unless machine is very weak.
Alternatively, validators can invoke the endpoint with
```
{
  address: '',
  // hash-onion locally created
  hashes: ['']
  distance: xxx,
  count: xxx,
}
```

**Review by shuse2 — APPROVED** [review]

**Pull request merged**